hello-world-master/source/components/scene/helloworld.brs

```diff
@@ -1,36 +1,35 @@
 sub init()
     m.imageRowList = m.top.findNode("imageRowList")
     m.button = m.top.findNode("button")
-    showImage()
     m.imageRowList.setFocus(true)
     m.button.observeField("buttonSelected", "onButtonSelected")
     m.imageRowList.observeField("rowItemSelected", "onRowItemSelected")
+    m.getRequestTask = CreateObject("roSGNode", "GetRequestTask")
+    m.getRequestTask.ObserveField("itemContent", "onFetchData")
+	  m.getRequestTask.control = "RUN"
+    m.newScreen = CreateObject("roSGNode", "DescriptionScreen")
+    m.newScreen.visible = false
+    m.top.appendChild(m.newScreen)
```

Comment:
We should avoid creating an invisible screen on top of the current screen at `init` time.
Can we define a function to `navigateToDescriptionScreen(screenContent)` that would:
1. Create the `DescriptionScreen` node;
2. Set it's content;
3. Append it over the current screen;
4. Set focus to the `DescriptionScreen`.

This function should be called when a rowItem is selected.